src/hex_http.erl

```diff
@@ -7,7 +7,9 @@
 
 -type method() :: get | post | put | patch | delete.
 -type status() :: non_neg_integer().
+-export_type([status/0]).
 -type headers() :: #{binary() => binary()}.
+-export_type([headers/0]).
```

Comment:
we'll be changing this type in the future (https://github.com/hexpm/hex_core/issues/77) but it's probably ok to export it since we use it elsewhere, otherwise we get the warnings.